### ISSUE TITLE
[BUG] Lost templates

### ISSUE THREAD

**Issue by acidric** [bostrot/wsl2-distro-manager#226]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi, I use Wsl distro manager with everything on D:\WSL2-Distros. Everything was working fine, until I had the problem: 'wsl2-distro-manager opens with a blank screen'.
I hadn't seen the solution proposed in github. So I uninstalled and reinstalled Wsl Distro Manager.
Everything works again, except that when I go to Templates, I can't see my templates even though they are in (ex:D:\WSL2-Distros\templates\kali-linux.ext4)
How can I get it to see them again?

### Expected Behavior

Be able to reimport my templates after a crash or correct the configuration file

### Environment

```markdown
- Windows Version:Windows 11 Pro 10.0.22635 N/A build 22635
- WSL Distro Manager Version:1.8.11.0
- WSL Version: 2.0.9.0
- Source: Winget
```

**Comment by bostrot** [comment]
I agree there should be an option to re-import them! Until then just create a new distro and use the template as a rootfs.

**Comment by acidric** [comment]
Thank you very much for your reply, but I'm not sure I know how to do this. I'll look into it.

**Comment by bostrot** [comment]
Use C:\WSL2-Distros\templates (or your custom path) when creating a new instance as the image.

**Comment by bostrot** [comment]
Fixed in #318 — the template list is now reconciled with the templates folder on every read, so any `.ext4` sitting there is picked up again even if the preferences file was lost. Sorry it took this long.
